metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/service/DatasetService.java

```diff
@@ -341,6 +358,10 @@ public DatasetXslt createDefaultXslt(MetisUser metisUser, String xsltString)
     authorizer.authorizeWriteDefaultXslt(metisUser);
     DatasetXslt datasetXslt = null;
     if (xsltString != null) {
+      final DatasetXslt latestDefaultXslt = datasetXsltDao.getLatestDefaultXslt();
+      if (latestDefaultXslt != null) {
+        cleanDatasetXslt(latestDefaultXslt.getId());
```

Comment:
Same as above.

Reply:
Yes the idea is that we should update those scripts to include the cleanup of xslts as well. The data for those are generally not that big or crucial, so we don't really need a daemon running every day. Even if we forget cleaning one time the data will be very minimal. Nonetheless we could include a cleanup during a reindexing operation or even an "extra" daemon in metis-core that could be performed monthly or so..